src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/SharedBlobListener.cs

```diff
@@ -91,14 +91,8 @@ public void Dispose()
 
         private static IBlobListenerStrategy CreateStrategy(IStorageAccount account)
         {
-            if (!StorageClient.IsDevelopmentStorageAccount(account))
-            {
-                return new PollLogsStrategy();
-            }
-            else
-            {
-                return new ScanContainersStrategy();
-            }
```

Comment:
Your new strategy references the existing PollLogsStrategy - it can't be removed